docs/use-mainnet/gas-on-linea.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Gas on Linea
+description: Understanding gas on Linea
+---
+
+## What does gas pay for?
+
+Gas must be paid to cover any kind of on-chain computation. For a zk rollup like Linea, gas mainly goes toward the cost of generating proofs for batches of transactions and submitting this data to L1.
+
+The main costs of interacting with L1 are:
+- Proof verification
+- Producing the hash of the batch submission data, which is required as an input to the Plonk verifier
+- Sending `calldata` to L1.
```

Comment:
Replace with: Publishing compressed transactions as 'calldata' to L1

Can remove the line about the hashing cost (it's only a minor cost comparatively)